packages/extension-base/src/services/balance-service/helpers/subscribe/substrate/index.ts

```diff
@@ -97,36 +97,47 @@ async function subscribeWithSystemAccountPallet (addresses: string[], chainInfo:
     }
 
     const items: BalanceItem[] = balances.map((balance: AccountInfo, index) => {
-      let total = balance.data?.free?.toBn() || new BN(0);
+      const free = balance.data?.free?.toBn() || new BN(0);
       const reserved = balance.data?.reserved?.toBn() || new BN(0);
       // @ts-ignore
       // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
-      const miscFrozen = balance.data?.miscFrozen?.toBn() || balance?.data?.frozen?.toBn() || new BN(0);
+      const locked = balance.data?.miscFrozen?.toBn() || balance?.data?.frozen?.toBn() || new BN(0);
```

Comment:
Need to check when it's `balance.data.frozen` or `balance.data.miscFrozen`

Reply:
Currently, I see only some chains: edgeware, joystream, terno have balance.data.miscFrozen. The common point of these chains is belong to "Live Networks" on Polkadot(.js) UI.

Reply:
Mark TODO on this line for later reviews